lib/ios-crash-log.js

```diff
@@ -0,0 +1,63 @@
+import path from 'path';
+
+import _ from 'lodash';
+import _glob from 'glob';
+import { fs, getLogger } from 'appium-support';
+import B from 'bluebird';
+let glob = B.promisify(_glob);
+
+
+require('date-utils');
+let logger = getLogger('IOS-crash-log');
+
+class IOSCrashLog { 
+  constructor() {
+    this.prevLogs = [];
+    this.logsSinceLastRequest = [];
+    this.crashDir = path.resolve(process.env.HOME, 'Library', 'Logs', 'DiagnosticReports');
+  }
+
+  async getCrashes () {
+    try {
+      return glob(`${this.crashDir} /*.crash`);
+    } catch (err) {
+      logger.errorAndThrow('There was a problem getting the crash list');
```

Comment:
`errorAndThrow` already throws, so no need to `throw err` again.

also, in cases like these, it's usually helpful to include the original error along with the nice message, like:

```
logger.errorAndThrow(`There was a problem getting the crash list: ${err}`);
```

That way we have at least something to go on in debugging